04/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
```

Comment:
Where are all meta tags we have seen? / Dónde están los meta tags que hemos visto?